prepare/metrics/string_containment.py

```diff
@@ -31,3 +31,36 @@
 )
 
 add_to_catalog(metric, "metrics.string_containment", overwrite=True)
+
+
+# Add metrics.string_containment.ratio
+
+reference_field = "entities"
+
+metric = StringContainmentRatio()
+
+instance_targets = [
+    {"string_containment": 0.75, "score": 0.75, "score_name": "string_containment"},
+    {"string_containment": 0.0, "score": 0.0, "score_name": "string_containment"},
+]
+
+global_target = {
+    "string_containment": 0.38,
+    "score": 0.38,
+    "score_name": "string_containment",
+    "score_ci_high": 0.75,
+    "score_ci_low": 0.0,
+    "string_containment_ci_high": 0.75,
+    "string_containment_ci_low": 0.0,
+}
+
+outputs = test_metric(
+    metric=metric,
+    predictions=predictions,
+    references=[["dummy"] for _ in references],
+    instance_targets=instance_targets,
+    global_target=global_target,
+    task_data=[{reference_field: w} for w in references],
+)
+
+add_to_catalog(metric, "metrics.string_containment.ratio", overwrite=True)
```

Comment:
```suggestion
add_to_catalog(metric, "metrics.string_containment_ratio", overwrite=True)
```

Seems it would be easier to find this way in the catalog (not nested).